app/models/template.rb

```diff
@@ -1,7 +1,8 @@
 class Template < ActiveRecord::Base
   validates_lengths_from_database
 
-  attr_accessible :name, :template, :snippet, :audit_comment, :location_ids, :organization_ids, :locked, :default
```

Comment:
Right, but I explicitly did https://github.com/theforeman/foreman/commit/bf5f18013ad224a93d917d696d94952965a8968d recently to be able to set locked at creation, it made rex seeds cleaner.  Why prevent mass assignment? The auth check would still happen.